extension/chrome/elements/pgp_block.ts

```diff
@@ -436,6 +453,28 @@ Catch.try(async () => {
     }
   };
 
+  const addSecondPart = (message: string, isHtml: boolean = false) => {
+    const pgpBlock = $("#pgp_block")
+      .append('<div id="action_open_full" class="three_dots"><img src="/img/svgs/three-dots.svg" /></div>'); // xss-safe-factory
+    if (isHtml) {
+      pgpBlock.append(`<div id="forwarded_msg" class="forwarded_msg" style="display:none">` + // xss-safe-factory
+        `${Xss.htmlSanitizeKeepBasicTags(message)}
+          </div>`);
+    } else {
+      pgpBlock.append(`<div id="forwarded_msg" class="forwarded_msg" style="display:none">` + // xss-safe-factory
+        `${Xss.htmlSanitizeKeepBasicTags(Xss.escapeTextAsRenderableHtml(message))}
+                  </div>`);
+    }
+    $('#action_open_full').click(Ui.event.handle(async target => {
+      if ($("#forwarded_msg").css('display') === 'none') {
+        $("#forwarded_msg").css('display', 'block');
```

Comment:
Could shorten these 5 lines to
```js
$("#forwarded_msg").css('display', $("#forwarded_msg").css('display') === 'none' ? 'block' : 'none');
```

Reply:
Yes, I forget about ternary operator because I don't often use it. But in this case, it looks great.
I will try to use it more often. Thanks!